src/Service/SegmentEventsService.php

```diff
@@ -94,6 +95,29 @@ function () use ($version, $limit, $page) {
         );
     }
 
+    public function findByProgrammeItemUsingOriginalVersion(
```

Comment:
I'm a bit confused about this method. What's happening? What's being passed in and what is being returned?

Reply:
Don't even bother. After talking to Bjorn, we've decided that just fetching all of the versions will be much easier.